### ISSUE TITLE
Window doesn't resize correctly with Resize Desktop option enabled on Gnome 40 on Wayland

### ISSUE THREAD

**Issue by SamLukeYes** [nick-shmyrev/improved-osk-gnome-ext#8]
When the keyboard pop up on Gnome 40 (Wayland), windows can't resize correctly with the desktop, and some parts of the window can be left under the keyboard. Sometimes they don't resize at all, but when I push the Activity button with the keyboard popped up, the desktop and windows look resized for a second, and then return to the original size as the keyboard goes down.

The resizing option works as expected on Xorg, but my touchscreen is almost unusable on Xorg compared with Wayland. The original extension worked as expected on Gnome 3.38 on Wayland, but also have this issue on Gnome 40.

**Comment by nick-shmyrev** [comment]
Yeah, I'm seeing this issue as well. I've played with it a bit and it seems to be somehow related to focus. I.e., it will resize the desktop after you initially focus on an input (and only if the keyboard would obscure that input when shown), but if you hide and then show the keyboard again without changing focus it will no longer resize desktop. But, if you change the focus to something else, focus on the same input again and tap it, keyboard will once again resize the desktop.

Also, it looks like the build-in Gnome keyboard has this issue too, so perhaps it originates somewhere in Gnome and/or Wayland?
